pkg/network/types.go

```diff
@@ -36,3 +36,15 @@ func (c CIDR) ToIPNet() *net.IPNet {
 	netw := net.IPNet(c)
 	return &netw
 }
+
+func (c *CIDR) IsIPv4() bool {
+	return c.IP.To4() != nil
+}
+
+func ParseIPNet(cidr string) CIDR {
+	_, prefix, err := net.ParseCIDR(cidr)
+	if err != nil {
+		panic(err)
+	}
+	return CIDR(*prefix)
+}
```

Comment:
Please do not use this in standard code. Having `panic(...)` in production code is usually not a good idea.
The code parsing the CIDRs actually has error handling. Hence, it should also check for errors.
For test code this is fine, though.